bench/workgen/runner/example_simple.py

```diff
@@ -41,19 +41,22 @@ def show(tname):
     print('<><><><><><><><><><><><>')
     c.close()
 
-context = Context()
-conn = wiredtiger_open("WT_TEST", "create,cache_size=1G")
-s = conn.open_session()
-tname = 'table:simple'
-s.create(tname, 'key_format=S,value_format=S')
+def example_simple_workload(args):
```

Comment:
Since this is an example, I'd add a comment about `args`.  Like `args` gives us access to command line arguments.  This example doesn't use any.

Reply:
I agree with @ddanderson. It will be nice to have an example use of input arguments other than `-K` option.

Reply:
f0e020f